### DISCUSSION TITLE
Welcome to emacs-tree-sitter Discussions!

### DISCUSSION THREAD

**Discussion by ubolonton** [emacs-tree-sitter/elisp-tree-sitter#126] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.
  * 📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by elzorrorebelde** [comment]
Jorge is my name. I try to help with issues and pull request in labeling and answering what I can. I'm the author of query-builder for emacs-tree-sitter (which has grown by itself thanks to you and the community)
